EarlGrey/Common/GREYTestHelper.h

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+/**
+ *  Provides interface for test helper methods.
+ */
+@interface GREYTestHelper : NSObject
+
+/**
+ *  Speeds up animations.
+ */
++ (void)increaseAnimationSpeed;
```

Comment:
Suggest renaming to enableFastAnimation and resetAnimationSpeed to disableFastAnimation (or something similar) as increaseAnimationSpeed sounds like I can call it multiple times to keep increasing the speed.